python/ingestion/cdc_wisqars_utils.py

```diff
@@ -104,17 +104,24 @@ def convert_columns_to_numeric(df: pd.DataFrame, columns_to_convert: List[str]):
         df[column] = pd.to_numeric(df[column], errors='coerce')
 
 
-def generate_cols_map(prefixes, suffix):
+def generate_cols_map(estimated_total_cols: List[str], suffix: str):
     """
-    Generates a mapping of columns with given prefixes to their corresponding columns
-    with specified suffix.
+    Generates a mapping of column names from a list of estimated total column names to a column name
+    with a specified suffix.
 
     Parameters:
-    prefixes (list): A list of prefixes to generate the mapping for.
-    suffix (str): The suffix to add to the columns.
+        estimated_total_cols (List[str]): A list of column names, each ending with a standard raw suffix.
+        suffix (str): The suffix that will replace each RAW_SUFFIX in the new column names.
```

Comment:
suffix (str): The new suffix that will applied to the incoming list of prefixes for the generated column names